servicetalk-opentracing-inmemory/src/main/java/io/servicetalk/opentracing/inmemory/CharSequenceKeyValueAccessor.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.opentracing.inmemory;
+
+import javax.annotation.Nullable;
+
+/**
+ * An Key Value accessor interface that works with {@link CharSequence} types.
+ */
+public interface CharSequenceKeyValueAccessor {
```

Comment:
nit: 

The typical pattern in open tracing repo is to call these `<myType>Extract` or `<myType>Inject` i.e. 

we would split this class up into two interfaces and call them `CharSequenceKeyValue(Extract|Inject)`. 

leaving the comment here in case we want to be consistent as users of open tracing may be familiar with that naming scheme.

Reply:
The classes are now gone. They B3KeyValue formatter allows for generic injector/extractors to be passed in.